src/stylesheets/components/_section.scss

```diff
@@ -2,17 +2,17 @@
   @include u-padding-y($site-margins);
 
   @include media($medium-screen) {
-    @include u-padding-y(6rem);
+    @include u-padding-y(units(6));
```

Comment:
Use the appropriate size listed in the conversion table #2660

Reply:
is that 8, in this case?

Reply:
Yep!